ui/apps/platform/src/Containers/Policies/Wizard/Form/descriptors.tsx

```diff
@@ -735,18 +735,18 @@ export const policyConfigurationDescriptor: Descriptor[] = [
     },
     {
         label: 'Allow privilege escalation',
-        name: 'Allow Privilege Escalation',
-        shortName: 'Allow Privilege Escalation',
-        longName: 'Privilege Escalation allowed on Container',
+        name: 'Allow privilege escalation',
+        shortName: 'Allow privilege escalation',
+        longName: 'Privilege escalation allowed on container',
         category: policyCriteriaCategories.CONTAINER_CONFIGURATION,
         type: 'radioGroup',
         radioButtons: [
             {
-                text: 'Allowed',
+                text: 'Privilege escalation allowed',
```

Comment:
Why did this go from Allowed to "Privilege escalation allowed"? The heading already says "Allow privilege escalation" so the wording seems redundant. 

@vjwilson or @pedrottimark thoughts on this from the UI side?

Reply:
Changed it to make the buttons look more descriptive, but yes it might look redundant too. Will wait from thoughts from UI to roll it back.

Reply:
I recommend changing the policy criteria title to `Privilege escalation`. Currently, the title indicates a value—allowed.

Reply:
@charmik-redhat 

Late to the party, but I just realized that a similar discussion is going on about another new policy criterion in another PR.

The UX team has come up with a suggested way forward there. If we copy their UI presentation, it would "allow" us to only present the option that makes sense for alerts. 

https://github.com/stackrox/stackrox/pull/1194#issuecomment-1090874152

TL;DR; When you drag the criterion into the policy, only show the alert-able state, Allow, as static text.

Reply:
Don't think the `RadioGroupDescriptor` allows to just keep a static text for `radioGroup` types (correct me if I am wrong), but just one radio button in options saying 'Allowed' and being always selected sounds like a good idea.

Reply:
@vjwilson, thanks for linking the PR. I looked at the PR  and the approach and I am not happy about it—the policy criteria title is myopic and it hinders extensibility. For instance, network policies are also risky (esp egress net pol). Extending support to alert on deployments impacted by network policies is equally useful. Now to support this, one would need to add separate policy criteria title, which seems unnecessary. Please feel free to share my feedback in your discussions or loop me in.

@charmik-redhat we should support both positive and negative cases because:
a. since they come free, so why the restriction?
b. System policies feature is the only way in our product to monitor env. These notifications could be for the purpose of monitoring security weakness, but also just monitoring the env.

Reply:
> @vjwilson, thanks for linking the PR. I looked at the PR and the approach and I am not happy about it—the policy criteria title is myopic and it hinders extensibility. For instance, network policies are also risky (esp egress net pol). Extending support to alert on deployments impacted by network policies is equally useful. Now to support this, one would need to add separate policy criteria title, which seems unnecessary. Please feel free to share my feedback in your discussions or loop me in.
> 
> @charmik-redhat we should support both positive and negative cases because: a. since they come free, so why the restriction? b. System policies feature is the only way in our product to monitor env. These notifications could be for the purpose of monitoring security weakness, but also just monitoring the env.

@md2119 You make good points. I'm copying @alanonthegit and @zhenpesky, so they see your feedback.

This is the sort of discussion I'd like to see "shift left", where we discuss problems and pros/cons of possible solutions before we get to the PR stage.

Reply:
@vjwilson @md2119 thanks for the context.

This discussion over network policies took place between 2 SEs, @boazmichaely, and @fredrb if I recall correctly. It was a recommendation by SE based on the general use case. There is always an extensibility vs practicality balance to find. We can follow up.